scss/civicrm/_mixins.scss

```diff
@@ -1,3 +1,11 @@
+/*
+* Use Open Sans as primary font.
+*/
+
+@mixin open_sans_font_family() {
+  font-family: 'Open Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif !important;
```

Comment:
Why it has to force the font? Why do we need `!important`?

Reply:
The this `mixin ` is expected to be used in places where font family might be already defined. We need `!important` here to force the use for the respective font family.